### PR TITLE
Add an editorconfig file.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+# Python configuration
+[*.py]
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
```
Comment (comzeradd) [inline .editorconfig]: You can have a `[*]` section for common things. (trailing white space, charset, space for indent). You can also add an `end_of_line = lf` just ot be safe.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+# Python configuration
+[*.py]
```
Comment (comzeradd) [inline .editorconfig]: You can also add a css/less here, where we also use 4 spaces.

**Comment by akatsoulas** [comment]
@comzeradd pr updated. Thanks for the review!

**Comment by comzeradd** [comment]
r+

**Pull request merged**